DSharpPlus/DiscordConfiguration.cs

```diff
@@ -86,7 +86,7 @@ public string Token
     /// <para>Disabling this option will increase the amount of traffic sent via WebSocket. Setting <see cref="GatewayCompressionLevel.Payload"/> will enable compression for READY and GUILD_CREATE payloads. Setting <see cref="Stream"/> will enable compression for the entire WebSocket stream, drastically reducing amount of traffic.</para>
     /// <para>Defaults to <see cref="Stream"/>.</para>

```

Comment:
incorrect, should be updated, potentially with an explanation as to why

Reply:
Whoops, right.